src/components/tests/QuickOpenModal.spec.js

```diff
@@ -163,4 +163,20 @@ describe("QuickOpenModal", () => {
       .simulate("change", { target: { value: "@someFunc" } });
     expect(filter).toHaveBeenCalledWith([], "someFunc", { key: "value" });
   });
+
+  test("Simple goto search query = :abc & searchType = goto", () => {
+    const { wrapper } = generateModal(
+      {
+        enabled: true,
+        query: ":abc",
+        searchType: "goto",
+        symbols: {
+          functions: [],
+          variables: []
+        }
+      },
+      "mount"
+    );
+    expect(wrapper).toMatchSnapshot();
```

Comment:
I'd add an assertion like `expect(wrapper.state().results).toEqual([])`

Reply:
add right underneath the toMatchSnapshot() line or replace it?

Reply:
Under it is best then you get two tests. 😄